moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -233,9 +244,34 @@ const DashboardView = () => {
                     }}
                   />
                 )}
-              </Box>
-            </div>
-          </div>
+              </Grid>
+            </Box>
+            <Box px={3} pb={3}>
+              <Card className={classes.cardWrapper}>
+                <CardContent>
+                  <Grid className={classes.cardTitle}>
+                    <Typography variant="h3" color="primary">
+                      Views
+                    </Typography>
+                  </Grid>
+                  <Grid item xs={12} sm={6} md={3}>
+                    <Card>
```

Comment:
I think this would work a lil nicer if the whole card where clickable instead of just the text

Reply:
Went ahead and made a commit with this small change: 
https://github.com/cityofaustin/atd-moped/pull/657/commits/2ccea44733670bc2a0ef2d73321945ec05428984